change-beta/@azure-communication-react-b433dc4b-66ce-46e2-bffb-2822d6a36a5a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
Should it be 'minor'?

Reply:
> Should it be 'minor'?

This PR is to add a change log and prepare this feature for release. That's why we keep the type as prerelease.